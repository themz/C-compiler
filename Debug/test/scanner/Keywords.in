continue
do
double
sizeof
else
for
if
int
return
struct
break
typedef
void
while
char