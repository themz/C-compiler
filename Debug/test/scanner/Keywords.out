ReservedWord		1		1		continue		
ReservedWord		2		1		do		
ReservedWord		3		1		double		
ReservedWord		4		1		sizeof		
ReservedWord		5		1		else		
ReservedWord		6		1		for		
ReservedWord		7		1		if		
ReservedWord		8		1		int		
ReservedWord		9		1		return		
ReservedWord		10		1		struct		
ReservedWord		11		1		break		
ReservedWord		12		1		typedef		
ReservedWord		13		1		void		
ReservedWord		14		1		while		
ReservedWord		15		1		char		
